src/main/java/org/fundacionjala/at13/spaceinvaders/Main.java

```diff
@@ -2,7 +2,7 @@
 
 public final class Main {
 
-    public static final int WAIT_PERIOD = 500; // milliseconds
+    public static final int WAIT_PERIOD = 50; // milliseconds
```

Comment:
Value of WAIT_PERIOD is too short.

Reply:
It was only changed to make a presentation, otherwise would take to much.